src/containers/search/SearchForm.tsx

```diff
@@ -35,8 +29,8 @@ import {
 import { formatISO, parseISO } from 'date-fns';
 import { SearchSchema } from './SearchScheme';
 import { convertToLocalTime } from 'src/utils/date';
-import { RHFAutocomplete } from 'src/components/hook-form/RHFAutocomplete';
-import { SearchPopovers } from './SearchPopovers';
+import { SearchPopovers, SearchPopoversProps } from './SearchPopovers';
```

Comment:
types and interfaces should be imported separately, before regular imports.eg. ```   import type { SearchPopoversProps } from './SearchPopovers'; ``` It makes code more readable